ebpf/session.go

```diff
@@ -691,10 +692,17 @@ func (s *session) linkKProbes() error {
 		prog     *ebpf.Program
 		required bool
 	}
-	var hooks = []hook{
+	var hooks []hook
+	archSys := ""
+	if "amd64" == runtime.GOARCH {
+		archSys = "__x64_"
+	} else {
+		archSys = "__arm64_"
+	}
```

Comment:
If we're not going to add support for some other architectures soon 😅

Reply:
i hope no